build.sbt

```diff
@@ -3,7 +3,7 @@ val V = new {
   val ScalaGroup = "2.13"
 
   val cats             = "2.4.1"
-  val laminar          = "0.11.0"
+  val laminar          = "0.12.2"
```

Comment:
Thanks!

Would you mind trying upgrading this to 0.13.0-M1 and see if it works? It's a recent release with small changes to Airstream so it'd be good if it gets all the testing it can.